docs/api/fhir-api-specification.md

```diff
@@ -2936,6 +2936,10 @@ On success, the server will return the newly created resource with an id. This i
   </div>
 </details>
 
+#### Duplicate Detection
+
+Duplicate detection can be enabled to prevent duplicate creattions, simply include the `If-None-Exist` header containing the search parameters. An example of this could be: `state-local-id=EX-286392birthDate=1966-12-01`
```

Comment:
I think this line needs a `&`

Reply:
creations is misspelled

Reply:
this example will need to be updated once the search params are modified